src/alchemlyb/preprocessing/subsampling.py

```diff
@@ -169,22 +191,27 @@ def u_nk2series(df, method='dE'):
             # for the state that is the last state, take the state-1
         else:
             series = df.iloc[:, index - 1]
-    else:  # pragma: no cover
+    else:
         raise ValueError(
             'Decorrelation method {} not found.'.format(method))
     return series
 
 
 @pass_attrs
-def dhdl2series(df, method=''):
+def dhdl2series(df, method='all'):
     """Convert a dhdl DataFrame to a series for subsampling.
 
+    The series is generated by summing over all energy components (axis 1 of
+    `df`), as for ``method='all'`` in :func:`u_nk2series`. Normally, `df` only
+    contains a single energy component.
```

Comment:
I think it might not be very good to have `Normally` in

> Normally, `df` only contains a single energy component.

If one use split protocol in Gromacs, the dhdl will always have more than one column and using the phrase `Normally` might indicate that having more than one column in dHdl is abnormal. I think it might be good to change it to 

> `Most `df` only contains a single energy component.

Reply:
thanks, reworded